Kconfig

```diff
@@ -21,13 +21,20 @@ config OV3660_SUPPORT
         Enable this option if you want to use the OV3360.
         Disable this option to safe memory.
     
-config SCCB_HARDWARE_I2C
-    bool "Use hardware I2C1 for SCCB"
-    default y
+choice SCCB_HARDWARE_I2C
```

Comment:
this should be added as separate setting that depends on hardware I2C being selected. It will remove the need of some of the ifdef stuff in the sccb.c file and show the setting only when hardware is selected.

Reply:
I understand your point here. Just pushed a commit to fix this, let me know about it.

Reply:
yup :) that is it! thanks a bunch!